sdk/core/core-amqp/package.json

```diff
@@ -67,7 +67,7 @@
   "dependencies": {
     "@azure/abort-controller": "^1.0.0",
     "@azure/core-auth": "^1.1.2",
-    "@azure/logger": "^1.0.0",
+    "@azure/logger": "^1.0.1",
```

Comment:
@sadasant, @KarishmaGhiya, There is no change to the logger after we shipped 1.0.0 more than half a year ago...
So, why would this change be required?

Reply:
The issue is with min/max tests. Our current code is not compatible with that older 1.0.0, as far as our current tests can confirm.

Reply:
But logger 1.0.1 is not released and as far as I can see is no different than 1.0.0 
Why would we be doing min/max testing with unreleased dependencies?

Reply:
Also I would prefer not to update to a version that we don't plan to ship yet anytime soon.

Reply:
I'm actually unable to replicate why logger 1.0.0 might be an issue 🤔 I'll report to Karishma. I changed this PR title to "DO NOT MERGE" for now.

Reply:
![image](https://user-images.githubusercontent.com/8968058/81125855-56551a80-8eee-11ea-8c08-e503491aa635.png)
Was getting these errors in all the tests - @HarshaNalluru and I were discussing about these last week and errors like these may be due to the mixed typings in the files coming from the local sources and package references. We definitely need to separate out the public facing tests and the internal references in separate folders for min-max testing to actually work otherwise this causes the typings to get mixed. 

Also I don't know if bumping the version is going to be of much help for core-logger

Reply:
Re-iterating from before, bumping the version of logger cannot possibly help because the logger has seen no changes after 1.0.0

@sadasant, Please update the PR to remove the logger related changes. The logger related error @KarishmaGhiya has listed above needs to be investigated separately

Reply:
Sounds good! Thank you

Reply:
This PR only affects `@azure/core-paging` now.